src/main/java/com/projetogerenciamentocurso/gerenciamentocurso/models/Aluno.java

```diff
@@ -7,6 +7,7 @@
 import javax.persistence.CascadeType;
 import javax.persistence.Entity;
 import javax.persistence.FetchType;
+import javax.persistence.Id;
 import javax.persistence.ManyToMany;
 
 import com.sun.istack.NotNull;

```

Comment:
Utilizar o NotNull do javax validation
```suggestion
import javax.validation.constraints.NotNull;
```

Reply:
feito